packages/astro/src/@types/astro.ts

```diff
@@ -666,6 +677,16 @@ export interface AstroUserConfig {
 	 * ```
 	 */
 
+	/**
+	 * @docs
+	 * @name server.headers
+	 * @typeraw {OutgoingHttpHeaders}
+	 * @default `{}`
+	 * @version 1.70.0
```

Comment:
```suggestion
	 * @version 1.7.0
```
Just checking on this. (also in line 328)